chore: update respect ajv setup to use provided context (#2860)
Redocly/redocly-cli: .changeset/eleven-clubs-trade.md

---
'@redocly/respect-core': patch
---

Fixed `respect` schema checks to honor `readOnly` and `writeOnly` based on context.

> **Warning:** `writeOnly` property that is present in a response is now reported as an error.

Reviewer: ```suggestion
**Warning:** `writeOnly` properties in responses are reported as errors.
```
